Dockerfile

```diff
@@ -1,16 +1,16 @@
-# Below version includes rust 1.76.0
-ARG RUST_IMAGE_VERSION=@sha256:2a6147ed8a879621f27d2c7a966868de438801084a276498b3a937e326ef7f65
+# Below version includes rust 1.80.1
```

Comment:
Personally, I'm happy with this, but if we're not doing a scrypto release, @iamyulong  wasn't sure we wanted to do this.

Honestly, given it works OK with 1.80, I'm happy to just update docs and image to suggest 1.80 - then if we want to, we can always consider publishing the new builder image if anyone complains. 👍

Reply:
I personally did this change since it was the only way to get CI to pass. Happy to keep it or to revert these changes. I don't see a problem with keeping it if we won't publish a new scrypto version and a new scrypto builder version.